components/MintNFT/index.tsx

```diff
@@ -45,7 +45,7 @@ class MintNFT extends React.Component<Props, State> {
 
   mint = async () => {
     const { referenceId, assetType, amount, tokenId } = this.state;
-    const registry = config.contractAddresses.COLLATERAL_NFT;
+    const registry = config.pools[0].addresses.COLLATERAL_NFT;
```

Comment:
Since `pool[0]` is hardcoded here – won't this imply that we might use the wrong registry if we are on another poll?

Reply:
yes, it is not ideal at all to hardcode it like this. 
The page is currently not exposed and is just used by us for testing purposes. I neeeded it to run for testing. I think we can also just hardcode the nftRegistry address in the code directly for now